taxi_service/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
Don't add files from the pycache and virtual env to the pull request, get rid of them.

Reply:
Got it.